src/backend/bottles/utils.ts

```diff
@@ -0,0 +1,114 @@
+import { spawn } from 'child_process'
+import { logDebug, LogPrefix } from '../logger/logger'
+import { isFlatpak } from '../constants'
+import shlex from 'shlex'
+
+function prepareBottlesCommand(
+  args: string[],
+  bottlesBin: string,
+  noCli?: boolean,
+  join?: boolean
+): string[] | string {
+  const command: string[] = []
+
+  if (isFlatpak) {
+    command.push(...['flatpak-spawn', '--host'])
+  }
+
+  if (bottlesBin === 'flatpak') {
+    command.push(...['flatpak', 'run'])
+    if (!noCli) {
+      command.push('--command=bottles-cli')
+    }
+    command.push('com.usebottles.bottles')
+  } else {
+    command.push('bottles-cli')
+  }
+
+  command.push(...args)
+  if (join) {
+    return shlex.join(command)
+  }
+  return command
+}
+
+async function getBottlesNames(
+  bottlesBin: 'flatpak' | 'os'
```

Comment:
You sometimes use `'flatpak' | 'os'` as your `bottlesBin`/`bottlesType`, and other times you just use `string`
Consider adding this type as an actual new `type` (either here or in `src/common/types.ts`) and referencing it everywhere, otherwise this is confusing for people who didn't write the code :^)